<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1496</width>
    <height>446</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="mainFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>361</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>451</width>
       <height>318</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="buttonLayout">
        <item>
         <widget class="QPushButton" name="startButton">
          <property name="text">
           <string>Power On</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="chargeButton">
          <property name="text">
           <string>Charge Battery</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="disconnectButton">
          <property name="text">
           <string>Disconnect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="topLayout">
        <item>
         <widget class="QProgressBar" name="batteryBar">
          <property name="styleSheet">
           <string notr="true">QProgressBar{ text-align:center; border: 2px solid grey; border-radius: 3px;} QProgressBar::chunk {background-color: #93C572; margin: 0.5px;}</string>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="format">
           <string>Battery: %p%</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="styleSheet">
           <string notr="true">QProgressBar{ text-align:center; border: 2px solid grey; border-radius: 3px;}  QProgressBar::chunk {background-color: #89CFF0;margin: 0.5px;;}  </string>
          </property>
          <property name="maximum">
           <number>300</number>
          </property>
          <property name="value">
           <number>300</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="format">
           <string>Cartridge: %vu</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="bolusButton">
        <property name="text">
         <string>Inject Bolus</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="glucoseLabel">
        <property name="text">
         <string>Current Glucose: 5.5 mmol/L</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="IOB">
        <property name="text">
         <string>Insulin On Board (IOB): 2.5u | 3:45 hrs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="logOutput">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="profileFrame">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>401</width>
      <height>361</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="profileHeaderLabel">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>11</y>
       <width>105</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Personal Profile:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createProfileButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>84</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Create New</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="defaultProfileLayout">
      <item>
       <widget class="QRadioButton" name="defaultProfileRadioButton">
        <property name="text">
         <string>   Regular</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="defaultProfileBasalSpinBox"/>
      </item>
      <item>
       <spacer name="spacer2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="deafultProfileGlucoseSpinBox"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>361</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="profileLabelLayout">
      <item>
       <widget class="QLabel" name="profileLabel">
        <property name="text">
         <string>Profile:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="basalLabel">
        <property name="text">
         <string>Basal Rate:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="targetGlucoseLabel">
        <property name="text">
         <string>Target Glucose:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>310</y>
       <width>168</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="ProfileButtonLayout">
      <item>
       <widget class="QPushButton" name="deleteProfileButton">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="submitProfileButton">
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1496</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
